CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+### Removed
+- Disable CallerProcedure header temporarily by stop propoagating this header in the outbound middleware.
```

Comment:
nit: can you use the exact name of the header (aka `rpc-caller-procedure`) as we did in the 1.53.0 changelog entry?